### PR TITLE
refactor: use Vite for VRT app

### PR DESCRIPTION
**Pull request by emmenko**
Same as https://github.com/commercetools/ui-kit/pull/2153

### CODE REVIEW

File: visual-testing-app/src/application.tsx [hunk 1]
```diff
@@ -1,72 +1,55 @@
-import { ComponentType, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+/// <reference types="vite/types/importMeta" />
```
Comment (CarlosCortizasCT) [inline visual-testing-app/src/application.tsx:1]: Is this line needed to use the `import.meta.globEager` function?

Comment (emmenko) [reply visual-testing-app/src/application.tsx]: Yes correct

File: visual-testing-app/src/application.tsx [hunk 2]
```diff
@@ -1,72 +1,55 @@
-import { ComponentType, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+/// <reference path="../../node_modules/vite/types/importMeta.d.ts" />
```
Comment (emmenko) [inline visual-testing-app/src/application.tsx:1]: FYI: we can also use

```js
/// <reference types="vite/client" />
```

But it contains additional declarations like for `*.svg` that conflicts with what we also use in the repo. So I opted to only import the type declarations for `import.meta`, but we need to use a relative import path for that.

**Review by CarlosCortizasCT — APPROVED** [review]
lgtm

**Pull request merged**